JustSaying.Tools/JustSaying.Tools.csproj

```diff
@@ -0,0 +1,25 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="..\build\common.props" />
+  <PropertyGroup>
+    <TargetFramework>net46</TargetFramework>
+  </PropertyGroup>
+  <ItemGroup>
+    <None Include="App.config" />
```

Comment:
I think these are still needed for "desktop" projects, according to dotnet sdk repo issues. tbh I'm still a little confused by all this, I think this is automagic on .NET Core but not for full framework, and should be on for OutputType=Exe (and I guess for test assemblies so the test runner can use them).